lib/ui/ui_utils.dart

```diff
@@ -32,40 +39,36 @@ PButton createExtensionButton(String text, String extensionName) {
   return button;
 }
 
-CoreElement createExtensionCheckBox(String extensionName) {
-  const String disabledTextColor = 'rgba(0, 0, 0, 0.3)';
+CoreElement createExtensionCheckBox(
+    ServiceExtensionDescription extensionDescription) {
+  final extensionName = extensionDescription.extension;
   final CoreElement input = checkbox();
-  final CoreElement text = span(text: extensionName);
 
-  // Disable checkbox for unavailable service extensions.
-  if (!serviceManager.serviceExtensionManager
-      .isServiceExtensionAvailable(extensionName)) {
-    input.disabled = true;
-    text.element.style.color = disabledTextColor;
-  }
-  serviceManager.serviceExtensionManager.hasServiceExtension(extensionName,
-      (available) {
-    input.disabled = !available;
-    text.element.style.color = available ? 'black' : disabledTextColor;
```

Comment:
Why did we get rid of the disabled text color?

Reply:
good catch. I lost this on a merge.
I've added back this functionality by tweaking the HTML slightly so all the label content is within a label tag and the CSS controls fading out the label and icon when disabled.